Crawler/CrawlerService.cs

```diff
@@ -1,95 +1,95 @@
+                Content = article.Content.Select(paragraph => lexer.GetTokens(paragraph).ToList()).ToList()
+            };
 
-			Console.WriteLine($"amount of numbers as words: {Normalize(numbersAsWords, wordsCount)}");
-            Console.WriteLine($"amount of numbers as digits: {Normalize(numbersAsDigits, wordsCount)}");
-            Console.WriteLine($"percentage of emotion words: {wordsAnalyzer.CalculatePercentageEmotionWords(allTokens)}");
-            Console.WriteLine($"amount of question words: {Normalize(questionWords, wordsCount)}");
+            foreach (var analyzer in analyzers)
+            {
+                DisplayResult(analyzer.Analyze(tokenizedArticle));
+            }
 
-            Console.WriteLine($"average words count between punctuation: {punctuationAnalyzer.CalculateAverageWordsCountBetweenPunctuation(allTokens)}");
-            Console.WriteLine($"max words count between punctuation: {punctuationAnalyzer.CalculateMaxWordsCountBetweenPunctuation(allTokens)}");
-            Console.WriteLine($"standard deviation of words count between punctuation: {punctuationAnalyzer.CalculateWordsCountBetweenPunctuationStandardDeviation(allTokens)}");
-            Console.WriteLine($"ninth decile words count between punctuation: {punctuationAnalyzer.CalculateWordsCountDecile(9, allTokens)}");
-            Console.WriteLine($"second decile words count between punctuation: {punctuationAnalyzer.CalculateWordsCountDecile(2, allTokens)}");
+            applicationLifetime.StopApplication();
+        }
 
-            Console.WriteLine($"count question marks: {Normalize(punctuationAnalyzer.CountCharacter('?', allTokens), wordsCount)}");
-            Console.WriteLine($"count exclamation marks: {Normalize(punctuationAnalyzer.CountCharacter('!', allTokens), wordsCount)}");
-            Console.WriteLine($"count colons: {Normalize(punctuationAnalyzer.CountCharacter(':', allTokens), wordsCount)}");
-            Console.WriteLine($"count dashes: {Normalize(punctuationAnalyzer.CountCharacter('-', allTokens), wordsCount)}");
-            Console.WriteLine($"count quotations mark: {Normalize(punctuationAnalyzer.CountCharacter('"', allTokens)/2 , wordsCount)}");
+        private void DisplayResult(AnalysisResult analysisResult)
+        {
+            var analysisResultAttr = analysisResult.GetType().GetCustomAttribute<AnalysisResultAttribute>();
+            if (analysisResultAttr != null)
+            {
+                Console.WriteLine($"===================={analysisResultAttr.ArticlePart}====================");
+            }
 
-			applicationLifetime.StopApplication();
+            foreach (var prop in analysisResult.GetType().GetProperties())
+            {
+                var resultAttr = prop.GetCustomAttribute<ResultAttribute>();
+                if (resultAttr != null)
+                {
+                    var value = double.Parse(prop.GetValue(analysisResult).ToString());
```

Comment:
Why always parse to double?

Reply:
for the display, int as double behaves like it should with the format 0.##
I wanted float but for some reason there are doubles in contentresult